src/Kdyby/Doctrine/EntityRepository.php

```diff
@@ -32,7 +32,7 @@
 class EntityRepository extends Doctrine\ORM\EntityRepository implements Persistence\QueryExecutor, Persistence\Queryable //, Persistence\ObjectFactory
 {
 
-	use \Kdyby\StrictObjects\Scream;
+	//use \Kdyby\StrictObjects\Scream;
```

Comment:
remove completely if impossible to use... also why?

Reply:
Kdyby\StrictObjects\Scream is not compatible with Doctrine\ORM\EntityRepository so the tests fail.
It's possible, that it will be in next versions.
See: https://github.com/Kdyby/StrictObjects/blob/master/src/InstanceMethodCallScreamer.php#L18
vs. https://github.com/doctrine/orm/blob/master/lib/Doctrine/ORM/EntityRepository.php#L175

I could also define __call(string $name, array $args) in Kdyby\Doctrine\EntityRepository and then just call parent::__call($name, $args), if you like such a solution more.

Reply:
hmm... just drop Scream I think...

Reply:
dropped